src/tools/annotation/CobbAngleTool.js

```diff
@@ -160,7 +159,7 @@ export default class CobbAngleTool extends BaseAnnotationTool {
 
     angle *= 180 / Math.PI;
 
-    data.rAngle = roundToDecimal(angle, 2);
+    data.rAngle = roundToDecimal(angle, 1);
```

Comment:
Why aren't there calculations for the uncertainty of this tool? Is it because it's an angle? Why is 1 passed instead of 2?

Reply:
According to the Diagnost's measurement uncertainty guideline, cobble angle is rounded at 0.1 degree without uncertainty estimation